slasher/db/indexed_attestations.go

```diff
@@ -67,6 +67,32 @@ func (db *Store) IndexedAttestation(targetEpoch uint64, validatorID uint64) ([]*
 	return iAtt, err
 }
 
+// DoubleVotes looks up in db for slashable data possibly multiple slashable actions were preformed by the same validator.
+func (db *Store) DoubleVotes(targetEpoch uint64, validatorIdx uint64, dataroot []byte, att *ethpb.IndexedAttestation) ([]*ethpb.AttesterSlashing, error) {
+	idxAttestations, err := db.IndexedAttestation(targetEpoch, validatorIdx)
+	if err != nil {
+		return nil, err
+	}
+	var iAtt []*ethpb.IndexedAttestation
```

Comment:
Prefer better variable naming. There's `idxAttestations` then `iAtt`. Which is which?

Reply:
thanks, done